### DISCUSSION TITLE
What is the correct way to get the full path in navigation ?

### DISCUSSION THREAD

**Discussion by dim-37** [unoplatform/uno.extensions#2358] [Q&A]
I tried Navigator.Route but it seems to become null under certain circumstances.

**Comment by DevTKSS** [comment]
@dim-37 from the Documentation, the answer would be IRouteNotifyer, BUT since I did try exactly this, there has been a issue created to this. only output I got was null or "TabBarItem" (string!) but never any realistic route

**Comment by DevTKSS** [comment]
see #2663  to this
